connector/solarwindsentityconnector/internal/storage/internal_storage.go

```diff
@@ -69,6 +70,7 @@ type internalStorage struct {
 	entityTtl                 time.Duration
 	ttlCleanUpIntervalSeconds time.Duration
 	logger                    *zap.Logger
+	keyBuilder                KeyBuilder
```

Comment:
```suggestion
	cacheKeyBuilder                KeyBuilder
```
maybe... so it is clear why some random keybuilder is used in internal storage.